### PR TITLE
Feat/1693 trigger to prevent over write

### LINKED ISSUE #1693: Add trigger to ETL pipeline to prevent over-writting of data
Issue:

The ETL pipeline is at risk of overwriting seedlot ownership quantity data what was added to the table after initial registration. 

Task:

We need to prevent the ETL from writing to the seedlot_owner_quantity table in oracle once the condition for this trigger, SEEDLOT.ORIGINAL_SEED_QTY > 0, is met.

Acceptance Criteria:

- [ ] Solution functions as outlined above and prevents over-writing
- [ ] Ready to Demo in Sprint Review
- [ ] Does what I have made have appropriate test coverage?
- [ ] Documentation and/or scientific documentation exists and can be found
- [ ] Peer Reviewed by 2 people on the team
- [ ] Manual testing of all PRs in Dev and Prod
- [ ] Merged

### PR DESCRIPTION
**Pull request by xiaopeng0202**
# Description
Closes #1693 

#### Changed
- if the ORIGINAL_SEED_QTY is larger than 0, we won't push SEEDLOT_OWNER_QUANTITY_EXTRACT into the pipeline

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

### CODE REVIEW

**Review by franTarkenton — APPROVED** [review]
Looks good to me

File: .gitignore [hunk 1]
```diff
@@ -208,4 +208,6 @@ sync/data/
 schemaspy/output
 
 # local testing
-*.bat
\ No newline at end of file
+*.bat
+
+oracle-api/config
```
Comment (mgaseta) [inline .gitignore:213]: What is this change for?

**Review by mgaseta — APPROVED** [review]
LGTM!
Added only one non-blocking question.

File: .gitignore (hunk 1 above)
Comment (DerekRoberts) [reply .gitignore:213]: That was a missing like break. Viscose can be set to catch those automatically.

**Comment by DerekRoberts** [comment]
Sorry @mgaseta, I accidentally deleted your comment!

**Pull request merged**